css/style.scss

```diff
@@ -648,10 +648,6 @@ video {
 	display: none;
 }
 
-.participantWithList li > a,
```

Comment:
are we sure it's not needed here?

Reply:
SHouldn't with Jan latest fixes on 14

Reply:
Yeah, and there’s probably more custom stuff that we can remove – but let’s do that case by case or when issues come up, like here. :)

Reply:
So on backport we need to keep this?